pinc/release_queue.inc

```diff
@@ -242,7 +243,8 @@ function _queue_data($row, $length = null, $unheld_length = null)
         "unheld_length" => (int)$unheld_length,
         "comment" => $row["comment"],
         "project_selector" => $row["project_selector"],
-        "release_criterion" => $row["release_criterion"],
+        "projects_target" => $row["projects_target"],
```

Comment:
We have to cast these as an int here (eg `unheld_length` above), otherwise they will be returned as a string in the JSON.

Reply:
Yup, should have noticed that.